README.md

```diff
@@ -13,31 +13,113 @@ allows conversion from previous versions.
 * libsdformat - The C++ parsing code contained within this repository,
   which can be used to read SDFormat files and return a C++ interface.
 
+[http://sdformat.org/](http://sdformat.org/)
+
 Test coverage:
 
 [![codecov](https://codecov.io/bb/osrf/sdformat/branch/default/graph/badge.svg)](https://codecov.io/bb/osrf/sdformat)
 
+# Installation
+
+We recommend following the Binary Installation instructions to get up and running as quickly and painlessly as possible.
+
+The Source Installation instructions should be used if you need the very latest software improvements, you need to modify the code, or you plan to make a contribution.
+
+## Binary Installation
+
+### Ubuntu
+
+On Ubuntu systems, `apt-get` can be used to install `sdformat`:
+```
+sudo apt install libsdformat<#>-dev libsdformat<#>
+```
+
+Be sure to replace `<#>` with a number value, such as 2 or 3, depending on
+which version you need, or leave it empty for version 1.
+
+### Windows
+
+Install [Conda package management system](https://docs.conda.io/projects/conda/en/latest/user-guide/install/download.html).
+Miniconda suffices.
+
+Create if necessary, and activate a Conda environment:
+```
+conda create -n ign-ws
+conda activate ign-ws
+```
+
+Install `sdformat`:
+```
+conda install libsdformat --channel conda-forge
```

Comment:
Interesting, isn't libsdformat versioned in conda? What version will this install?

Reply:
It is, it's just not in the package name:
```
>conda search libsdformat -c conda-forge
Loading channels: done
# Name                       Version           Build  Channel
libsdformat                    6.2.0      h21ff451_1  conda-forge
libsdformat                    6.2.0      h21ff451_2  conda-forge
libsdformat                    6.2.0      h21ff451_3  conda-forge
libsdformat                    6.2.0      h7dee32a_0  conda-forge
libsdformat                    9.1.0      h21ff451_0  conda-forge
libsdformat                    9.1.0      h21ff451_1  conda-forge
libsdformat                    9.1.0      h7dee32a_2  conda-forge
libsdformat                    9.3.0      h4bf3a07_1  conda-forge
libsdformat                    9.3.0      h4bf3a07_2  conda-forge
libsdformat                    9.3.0      hbeba28f_1  conda-forge
libsdformat                    9.3.0      hbeba28f_2  conda-forge
```
I think they default to the newest.

Reply:
Clarified in f38552c